README.md

```diff
@@ -37,15 +37,12 @@
 ___BOLT:___
 
 
-> git clone https://github.com/facebookincubator/BOLT llvm-bolt
-
-> mkdir build
-
-> cd build
-
-> cmake -G Ninja ../llvm-bolt/llvm -DLLVM_TARGETS_TO_BUILD="X86;AArch64" -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_ASSERTIONS=ON -DLLVM_ENABLE_PROJECTS="clang;lld;bolt"
-
-> ninja bolt
+git clone --depth=1 https://github.com/llvm/llvm-project.git
+cd llvm-project
+mkdir build
+cd build
+cmake -G Ninja ../llvm -DLLVM_TARGETS_TO_BUILD="X86;AArch64" -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_ASSERTIONS=ON -DLLVM_USE_LINKER=mold -DLLVM_ENABLE_PROJECTS="clang;lld;bolt"
+ninja -j24
```

Comment:
Sorry I did run it locally and just copy pasted, remove:
`-DLLVM_USE_LINKER=mold`
and change:
`ninja -j$(nproc)`